index.html

```diff
@@ -0,0 +1,271 @@
+          <a href="#" class="see1">See Project</a>
+        </section>
+      </section>
+    </section>
+    <section class="card-works card5">
+      <section class="mask">
+        <section class="img-placeholder b">
+          <section class="div">
+            <section class="right-works">
+              <h2 class="pro pro2">Data Dashboard Healthcare</h2>
+              <h2 class="pro pro1">Profesional Art Printing Data</h2>
+              <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+              <ul class="tags">   
+                <li class="tags1"><h6>html</h6></li>                                            
+                <li class="tags2"><h6>bootstrap</h6></li>
+                <li class="tags3"><h6>Ruby</h6></li>
+              </ul>
+            </section>
+          </section>
+        </section>
+      </section>
+      <section class="action">
+        <section class="but">
+          <a href="#" class="see1" id="see"  onclick="testing()">See Project</a>
+        </section>
+      </section>
+    </section>
+    <section class="card-works card6">
+      <section class="mask">
+        <section class="img-placeholder c">
+          <section class="div">
+            <section class="right-works">
+              <h2 class="pro pro1">Profesional Art Printing Data</h2>
+              <h2 class="pro pro2">Website Protfolio </h2>
+              <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+              <ul class="tags">   
+                <li class="tags1"><h6>html</h6></li>                                            
+                <li class="tags2"><h6>bootstrap</h6></li>
+                <li class="tags3"><h6>Ruby</h6></li>
+              </ul>
+            </section>
+          </section>
+        </section>
+      </section>
+      <section class="action">
+        <section class="but">
+          <a href="#" class="see1" id="see">See Project</a>
+        </section>
+      </section>
+    </section>
+  </section>
+</section>
+  <section class="about" id="about">
+    <div class="about-box">
+      <div class="about-sub">
+        <h1 class="primary">About Me</h1>
+        <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <section class="button">
+          <a href="#" class="get">Get My Resume</a>
+        </section>
+      </div>
+      <div class="img-aboutbox">
+        <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=" Diagram illustration 4"></section>
+        <!-- <img src="./images/illustration-about me@2x.png" alt="Diagram illustration 4 mobile version" class="img-aboutbig"> -->
```

Comment:
- [ OPTIONAL] I would advise you to remove all unused commented codes from this file. Removing unused commented code makes your code look tidy. 👍